modules/backend/assets/js/october.tabformexpandcontrols.js

```diff
@@ -160,4 +160,114 @@
     }
 
     $.oc.tabFormExpandControls = TabFormExpandControls
-}(window.jQuery);
\ No newline at end of file
+
+}(window.jQuery);
+
+/*
+ * Auto update WAI-ARIA when a user updates the tabs
+ */
+ $(document).ready(function() {
+
+     /* If no selected => select first */
+     if ($('.master-tabs a[aria-selected]').length === 0) {
+         $('.master-tabs a:first').attr({
+             'aria-selected': 'true',
+             'tabindex': '0'
+         });
+     } else if ($('.primary-tabs a[aria-selected]').length === 0) {
+         $('.primary-tabs a:first').attr({
+             'aria-selected': 'true',
+             'tabindex': '0'
+         });
+     } else if ($('.secondary-tabs a[aria-selected]').length === 0) {
+         $('.secondary-tabs a:first').attr({
+             'aria-selected': 'true',
+             'tabindex': '0'
+         });
+     } else if ($('.content-tabs a[aria-selected]').length === 0) {
+         $('.content-tabs a:first').attr({
+             'aria-selected': 'true',
+             'tabindex': '0'
+         });
+     }
+     // else if ($('.vertical-tabs a[aria-selected]').length === 0) {
+     //     $('.vertical-tabs a:first').attr({'aria-selected':'true', 'tabindex':'0'});
+     //     $('.vertical-tabs ul').attr('aria-orientation':'vertical');
+     // }
+
+     /* Click on a link */
+     $('body').on('click', '.master-tabs,.primary-tabs,.secondary-tabs,.content-tabs', function(event) { // Can add ,.vertical-tabs
+
+         let $target = $(event.target);
+         let tabNameClick = '';
+
+         if ($target.hasClass('master-tabs')) {
+             tabNameClick = '.master-tabs';
+         } else if ($target.hasClass('primary-tabs')) {
+             tabNameClick = '.primary-tabs';
+         } else if ($target.hasClass('secondary-tabs')) {
+             tabNameClick = '.secondary-tabs';
+         } else if ($target.hasClass('content-tabs')) {
+             tabNameClick = '.content-tabs';
+         }
+         // else if($target.hasClass('vertical-tabs')) {
+         //     tabNameClick = '.vertical-tabs';
+         // }
+
+         // remove aria selected on all links + remove focusable
+         $(tabNameClick + ' a').attr('aria-selected', 'false');
+
+         // add aria selected on $this + focusable
+         $(tabNameClick + ' a.focus-visible').attr('aria-selected', 'true');
+
+         // Important - Must be set to true for October to set class="active" in the <li>
+         return true;
+     });
+
+     /* Keyboard arrow selection */
+     $('body').on('keydown', '.primary-tabs,.secondary-tabs,.master-tabs,.content-tabs', function(event) { // Can add ,.vertical-tabs
```

Comment:
Same here:
`$('body').on('keydown', '.primary-tabs,.secondary-tabs,.master-tabs,.content-tabs', function(event) { // Can add ,.vertical-tabs`
to
`$('.primary-tabs,.secondary-tabs,.master-tabs,.content-tabs').on('keydown', function(event) { // Can add ,.vertical-tabs`

Reply:
Same comment about dynamic dom element event binding